### DISCUSSION TITLE
AI也能踢足球!?

### DISCUSSION THREAD

**Discussion by 110202015** [1101-GS3073-C/Python-Community-GS3073-C#67] [科技新知]
突然想到，如果有可以下棋的AI，那會不會有可以參與其他競技項目的AI呢?抱著這樣的想法，在網路搜尋了一陣子之後，找到了這篇文章。文章中說到，有個比賽叫做「機器人足球世界杯」，參與其中的科學家目的之一就是要讓機器人能在2050年時和世界盃冠軍隊一較高下。但文章中也有說，這個目標以現今的技術來說，還有很大的進步空間，因為其實足球不只需要運動能力，也需要智能、判斷力和默契，而這些問題不知何時才能解決。
文章連結：https://www.bbc.com/zhongwen/trad/science-58885123

**Comment by Timmy110403540** [comment]
感覺以後各式各樣的機器人比賽都有可能發生 XD

**Comment by cobb110202008** [comment]
AI的學習力已經遠超我們所能想像的了

**Comment by pocper1** [comment]
智能、判斷力和默契，機器人的智能一定夠，判斷力可以用model training 去整理過去的比賽資料判斷當前該做甚麼事情；默契，可以用無線通訊技術即時告知其他機器人要去哪個位置接球。

**Comment by briankuo123** [comment]
以後是不是會像寶可夢一樣會有各種競賽之類的然後看誰練的機器人比較厲害!

**Comment by wh20502** [comment]
我無法想像機器人與世界隊的人比賽時，萬一有什麼閃失，機器人會不會踢斷人的腿...

**Comment by minniewei** [comment]
在未來全球性比賽上可能就變成人與機器人一起組隊了

**Comment by 110409538** [comment]
那以後會不會有機器馬的賽馬比賽出現😂

**Comment by yasklop** [comment]
以後會不會有機器人的奧運會，感覺很酷

**Comment by ghost** [comment]
2050年我都50多歲了XD50多歲跟著別人熱血沸騰感覺很好笑XD

**Comment by Shiba010** [comment]
感覺以後如果舉辦人類對戰機器人的足球比賽，應該會是機器人贏。
